dev-packages/application-manager/README.md

```diff
@@ -0,0 +1,25 @@
+<div align='center'>
+
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>THEIA - APPLICATION-MANAGER EXTENSION</h2>
```

Comment:
it is not an extension, but a package providing API to build and start applications. The same for the application package and cli.

Reply:
> it is not an extension, but a package providing API to build and start applications. The same for the application package and cli.

Should I just drop `extension` as part of the name?